examples/private_set_intersection/client/private_set_intersection.cc

```diff
@@ -80,9 +80,13 @@ int main(int argc, char** argv) {
   ::oak::NodeClient::InitializeAssertionAuthorities();
 
   // Connect to the newly created Oak Node from different clients.
-  auto stub_0 = PrivateSetIntersection::NewStub(::grpc::CreateChannel(
+  auto channel_0 = ::grpc::CreateChannel(
       addr.str(),
-      ::asylo::EnclaveChannelCredentials(::asylo::BidirectionalNullCredentialsOptions())));
+      ::asylo::EnclaveChannelCredentials(::asylo::BidirectionalNullCredentialsOptions()));
+  auto node_client_0 = ::oak::NodeClient(channel_0);
+  auto attestation = node_client_0.GetAttestation();
```

Comment:
Similarly here, unless these types are complicated/verbose it would be more clear to make them explicit.

Reply:
Done, though coming from Rust this seems unnecessarily verbose.